prow/scripts/resources/compass-gke-overrides.tpl.yaml

```diff
@@ -0,0 +1,16 @@
+global:
```

Comment:
Why don't you add these overrides to the file in the compass repo?

Otherwise we will need to create a PR in infra when we want to change some overrides regarding benchmark job.

It would require envsub and passing variables like before anyways due to the issue with the jwks url. It is kind of hidden here as overrides are split between two files in two different repos.

Reply:
Most of the fields in that file depend on vars that are available in the script here and I guess they won't be changed once properly adjusted. 

The `migratorJob` field is on purpose here during the validation of the job. Once I verify it passes, I will move it to compass repo for sure.

Reply:
Those vars are already exported in the environment (in contrast to the ones for the jwks that needs to be explicitly passed), therefore you will not need to pass them to envsub (similarly to how this file is substituted). 

I think that it would be better to have just one file with overrides and get the whole picture from it.

I am ok also if you think that the separation between cluster specific and product specific overrides make sense. I don't know why I clicked request changes, I will dismiss the review, it was more of an ad-hoc comment that a review.